### PR TITLE
Navigation Sizing Changes

### PR DESCRIPTION
**Pull request by tokonoma**
Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed
 - What changes did you make to satisfy the AC?
     > Outline changes to user flows, any surprising architectural decisions 
     > Alignment between PR commits and this section helps your reviewers.

### How To Test
 - Describe any special dev environment setup required.
     > Anything outside basic hamurai-managed stuff counts as "special".
 - Describe any special accounts that need to be tested.
     > Include any specific user roles and necessary feature flags.

### To Do
- [ ] Any outstanding tasks go here
    > Manage reviewers' expectations on missing features, tests, bugs, late design changes, ...)
- [ ] Tag new routes in Pendo

### CODE REVIEW

**Comment by nicklemmon** [comment]
@tokonoma You'll need to actually make a branch of the project vs. a fork - is there a way to do that in the UI?

If you're having trouble getting git to work as a CLI, you can always use this:
https://desktop.github.com/

**Comment by tokonoma** [comment]
@nicklemmon I think I got my issues sorted out pushing from my local, let me try again!

**Comment by nicklemmon** [comment]
Duplicate of #1694 - closing.
